tests/Dockerfile

```diff
@@ -8,4 +8,5 @@ WORKDIR /test
 
 # Install datadog-lambda with dev dependencies from local
 COPY . .
-RUN pip install .[dev]
\ No newline at end of file
+RUN pip install .[dev]
+RUN pip install --find-links=https://s3.amazonaws.com/pypi.datadoghq.com/trace-dev/index.html ddtrace==0.35.1.dev5+g1e11b2dd -t .[dev]
```

Comment:
Should we use ddtrace 0.36.0?

Reply:
Oh, I forgot to remove this from the test docker file. This line was just temporary until the ddtrace-py release came out